src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -370,6 +370,7 @@ void mp::QemuVirtualMachine::suspend()
             update_state();
 
             update_shutdown_status = false;
+            vm_process->waitForFinished();
```

Comment:
No parallel in other backends?

Reply:
Not the `libvirt` backend since all operations go through `libvirtd`, there is really nothing to wait on.

Reply:
Ack